.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: Test
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6
+        architecture: x64
+    # This is from https://github.com/elastic/elastic-github-actions/tree/master/elasticsearch
```

Comment:
What about using something like https://github.com/marketplace/actions/setup-elasticsearch  ?

Reply:
Ah, didn't think to check unofficial ones. I checked the official one and that says "DONT USE THIS YET". https://github.com/elastic/elastic-github-actions/tree/master/elasticsearch . I wouldn't rush to use an unofficial one myself.

Reply:
Fair enough